lib/libc/net/lib_inetntop.c

```diff
@@ -114,7 +114,7 @@ static int inet_ipv4_ntop(FAR const void *src, FAR char *dest, socklen_t size)
 	}
 
 	ptr = (FAR uint8_t *)src;
-	sprintf(dest, "%u.%u.%u.%u", ptr[0], ptr[1], ptr[2], ptr[3]);
+	snprintf(dest, INET_ADDRSTRLEN, "%u.%u.%u.%u", ptr[0], ptr[1], ptr[2], ptr[3]);
```

Comment:
what is the correct buffer size?  "socklen_t size" or "INET_ADDRSTRLEN"

Reply:
yes it's correct. refer below description
*   dest - The dest argument points to a buffer where the function stores
 *          the resulting text string; it shall not be NULL.
 *   size - The size argument specifies the size of this buffer, which must
 *          be large enough to hold the text string (INET_ADDRSTRLEN
 *          characters for IPv4, INET6_ADDRSTRLEN characters for IPv6).